lib/livebook_web/live/output/input_component.ex

```diff
@@ -145,62 +134,60 @@ defmodule LivebookWeb.Output.InputComponent do
 
   defp html_input_type(:number), do: "number"
   defp html_input_type(:color), do: "color"
-  defp html_input_type(:url), do: "text"
+  defp html_input_type(:url), do: "url"
   defp html_input_type(:text), do: "text"
 
+  # TODO: revisit once phx-debounce doesn't defer phx-blur
```

Comment:
For me there is no reason to revisit, I am happy with things as is, but your call!